Logger/DictToDB.py

```diff
@@ -0,0 +1,63 @@
+import sqlite3
+
+
+class DictToDB:
+
+    __connection = None
```

Comment:
Those members should be owned by the instance not the classe. Consider more than one db and this wouldnt work anymore